.eslintrc.json

```diff
@@ -26,6 +26,12 @@
     }
   },
   "rules": {
+    "header/header": [
+      2,
+      "line",
+      " Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.\n See LICENSE.txt for license information.",
+      2
```

Comment:
Most of this is copied from the shared config, but this line is the one that makes it so that two newlines are required after the header